kernel/README.md

```diff
@@ -24,15 +24,87 @@ in the kernel source code. You can modify it as needed.
 After the kernel source code is ready it is possible to build the kernel.
 
 ```bash
-./build-kernel.sh build
+$ ./build-kernel.sh build
 ```
 
 
 ## Install the Kernel in the default path for Kata
 
 Kata Containers uses some default path to search a kernel to boot. To install
 on this path, the following command will install it to the default Kata
-containers path.
+containers path.  
```

Comment:
Will be useful to mention the actual default path as well.

Reply:
done :)